Explorer/Assets/Scripts/ECS/StreamableLoading/NftShapes/GetNftShapeIntention.cs

```diff
@@ -0,0 +1,33 @@
+using ECS.StreamableLoading.Common.Components;
+using ECS.StreamableLoading.NftShapes.Urns;
+using System;
+using System.Threading;
+
+namespace ECS.StreamableLoading.NftShapes
+{
+    public struct GetNftShapeIntention : ILoadingIntention, IEquatable<GetNftShapeIntention>
+    {
+        public readonly string URN;
+
+        public CommonLoadingArguments CommonArguments { get; set; }
+
+        public readonly CancellationTokenSource CancellationTokenSource => CommonArguments.CancellationTokenSource;
+
+        public GetNftShapeIntention(string urn, IUrnSource urnSource)
+        {
+            this.URN = urn;
+            CommonArguments = new CommonLoadingArguments(urnSource.UrlOrEmpty(urn));
+        }
+
+        public bool Equals(GetNftShapeIntention other) =>
+            URN == other.URN
+            && CommonArguments.Equals(other.CommonArguments)
+            && CancellationTokenSource.Equals(other.CancellationTokenSource);
```

Comment:
I should not compare `CancellationTokenSource`

Reply:
I have replaced it with Url equality